app/script/view_model/content/InputBarViewModel.js

```diff
@@ -79,30 +78,7 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
     this.hasFocus = ko.pureComputed(() => this.isEditing() || this.conversationHasFocus()).extend({notify: 'always'});
     this.hasTextInput = ko.pureComputed(() => this.input().length);
 
-    this.input = ko.pureComputed({
-      read: () => {
-        if (this.isEditing()) {
-          return this.editInput();
-        }
-
-        const textInput = this.conversationEntity() && this.conversationEntity().input().text;
-        return textInput || '';
-      },
-      write: value => {
-        if (this.isEditing()) {
-          return this.editInput(value);
-        }
-
-        if (this.conversationEntity()) {
-          const mentions = this.currentMentions();
-
-          this.conversationEntity().input({
```

Comment:
This was the main pain point that I wanted to remove. The circular dependency between `conv.input` and `inputBarViewModel.input`